.gitignore

```diff
@@ -1,6 +1,7 @@
 /build
 /kcov_build
 /target
+.vscode
```

Comment:
It's better to split this line into another patch.

Reply:
Fixed